ivy/functional/ivy/experimental/activations.py

```diff
@@ -458,3 +458,35 @@ def elu(
     }
     """
     return current_backend(x).elu(x, alpha=alpha, out=out)
+
+def sequence_length(
```

Comment:
not sure why this is added

Reply:
> not sure why this is added
@Infrared1029 
This was an edit to an older commit, I'm not sure why it merged both my pull requests into one.